src/lib/components/consignments/index.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from "react"
+import * as Relay from "react-relay"
+
+import {
+  LayoutChangeEvent,
+  NavigatorIOS,
+  ScrollView,
+  Text,
+  TouchableHighlight,
+  View,
+  ViewProperties,
+} from "react-native"
+
+/** The metadata for a consigned work */
+interface ConsignmentMetadata {
+  title: string | null
+  year: string | null
+  category: string | null
+  materials: string | null
+  width: number | null
+  height: number | null
+  depth: number | null
+  units: "in" | "cm"
+}
```

Comment:
I started a conversation about nullable entries [in our JS Slack](https://artsy.slack.com/archives/C2TQ4PT8R/p1495022464258847)